### ISSUE TITLE
Add custom element to readme

### ISSUE THREAD

**Issue by lekoala** [inorganik/countUp.js#325]
Hi there,

Great library!

If you are interested to add it to the readme, I created a custom element that wraps your library

https://github.com/lekoala/formidable-elements/blob/master/demo/count-up.html

Some points you might find interesting:
- automated usage of intersection observer (i think this replace nicely the scrollspy feature that is more complex that needed)
- option to use Intl.NumberFormat is wanted (something hard to achieve in pure js)
- auto detect decimals (for simple cases)

**Comment by inorganik** [comment]
Those are some nice enhancements, and Formidable elements looks like a nice library! Looks like they are web components and framework agnostic?

I'm happy to include this in the README, but I would like to give the user a little more context around usage, if they are like me and have never used custom elements/web components. I'd appreciate any extra context you can give. Thanks!

**Comment by lekoala** [comment]
yes the idea is to have standalone, autonomous components. currently since it's easier to maintain i have everything under one repo. it has no stable release yet because i'm still deciding if each component needs to have its own repo but that seems like a lot of maintenance :)

fully agree with you it may need a bit more context, i'll let you know once its done

**Comment by lekoala** [comment]
Here is the documentation

https://github.com/lekoala/formidable-elements/blob/master/docs/count-up.md

And a demo

https://codepen.io/lekoalabe/pen/MWxJoGN

**Comment by inorganik** [comment]
Thanks for that quick turnaround. I was thinking of adding a link to that readme under the "use countUp with" bulleted list. What should I call it - does this look ok?

- [custom element](https://github.com/lekoala/formidable-elements/blob/master/docs/count-up.md)

**Comment by lekoala** [comment]
looking great !

**Comment by inorganik** [comment]
README updated. Sorry for the delay.
